python/servo/testing_commands.py

```diff
@@ -161,12 +161,17 @@ def suite_for_path(self, path_arg):
     @Command('test-perf',
              description='Run the page load performance test',
              category='testing')
-    def test_perf(self):
+    @CommandArgument('--submit', default=False, action="store_true",
+                     help="submit the data to perfherder")
+    def test_perf(self, submit=False):
         self.set_software_rendering_env(True)
 
         self.ensure_bootstrapped()
         env = self.build_env()
-        return call(["bash", "test_perf.sh"],
+        cmd = ["bash", "test_perf.sh"]
+        if submit:
+            cmd += ["--submit"]
```

Comment:
We should check that the Treeherder environment variables are set if `submit` is True before we actually run the command, to improve the user experience (e.g. if you don't read carefully).